diff --git a/compiler/rustc_middle/src/ty/layout.rs b/compiler/rustc_middle/src/ty/layout.rs
index 53706cc8..a80a6693 100644
--- a/compiler/rustc_middle/src/ty/layout.rs
+++ b/compiler/rustc_middle/src/ty/layout.rs
@@ -1252,6 +1252,7 @@ pub fn fn_can_unwind(tcx: TyCtxt<'_>, fn_def_id: Option<DefId>, abi: ExternAbi)
     use ExternAbi::*;
     match abi {
         C { unwind }
+        | Cpp { unwind }
         | System { unwind }
         | Cdecl { unwind }
         | Stdcall { unwind }
