diff --git a/compiler/rustc_hir_typeck/src/callee.rs b/compiler/rustc_hir_typeck/src/callee.rs
index e5fcc68d..ff866b9c 100644
--- a/compiler/rustc_hir_typeck/src/callee.rs
+++ b/compiler/rustc_hir_typeck/src/callee.rs
@@ -195,6 +195,7 @@ pub(crate) fn check_call_abi(&self, abi: ExternAbi, span: Span) {
             }
 
             CanonAbi::C
+            | CanonAbi::Cpp
             | CanonAbi::Rust
             | CanonAbi::RustCold
             | CanonAbi::RustPreserveNone
